wodles/aws/aws_s3.py

```diff
@@ -2254,6 +2288,14 @@ def reformat_msg(self, event):
             AWSBucket.reformat_msg(self, event)
             yield event
 
+    def load_information_from_file(self, log_key):
+        if log_key.split('.', 1)[1] == 'jsonl.gz':
+            with self.decompress_file(log_key=log_key) as f:
+                json_list = list(f)
+                return [dict(json.loads(x), source=json.loads(x)['service']['serviceName']) for x in json_list]
```

Comment:
You are invoking `json.load` 2 times for each event here. This is not efficient.

Reply:
Changed using a variable with the `:=` operator